service/src/main/java/uk/gov/hmcts/reform/fpl/enums/MandatoryDocuments.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum MandatoryDocuments {
+    SOCIAL_WORK_CHRONOLOGY("socialWorkChronologyDocument", "1"),
+    SOCIAL_WORK_STATEMENT("socialWorkStatementDocument", "2"),
+    SOCIAL_WORK_ASSESSMENT("socialWorkAssessmentDocument", "3"),
+    SOCIAL_WORK_CARE_PLAN("socialWorkCarePlanDocument", "4"),
+    SOCIAL_WORK_EVIDENCE_TEMPLATE("socialWorkEvidenceTemplateDocument", "5"),
+    THRESHOLD("thresholdDocument", "6"),
+    CHECKLIST("checklistDocument", "7");
+
+    private final String propertyKey;
+    private final String interfaceDisplayOrder;
+
+    MandatoryDocuments(String propertyKey, String interfaceDisplayOrder) {
```

Comment:
Does the interfaceDeisplay order needs to be String?

Reply:
Possibly not but If we used int, we'd still need to covert to String to render the error message. Thoughts?

Reply:
I believe you should use correct data element as long as possible. I think you should be able to pass int to the String format or whenever you producing the error message as %d probably.